enterprise/api/v1/views.py

```diff
@@ -116,6 +116,21 @@ class EnterpriseCustomerViewSet(EnterpriseReadWriteModelViewSet):
     filter_fields = FIELDS
     ordering_fields = FIELDS
 
+    def get_serializer_class(self):
+        if self.action == 'basic_list':
+            return serializers.EnterpriseCustomerBasicSerializer
+        return self.serializer_class
+
+    @list_route()
+    # pylint: disable=invalid-name,unused-argument
+    def basic_list(self, request, *arg, **kwargs):
+        """
+            Enterprise Customer's Basic data list without pagination
```

Comment:
`without pagination`? I think it has pagination

Reply:
code that is responsible for paginating is not included here
```
page = self.paginate_queryset(queryset)
        if page is not None:
            serializer = self.get_serializer(page, many=True)
            return self.get_paginated_response(serializer.data)
```
Default `list` endpoint is returning  response like this.
```
{
    "count": 3,
    "num_pages": 1,
    "current_page": 1,
    "results": [
        {
            "uuid": "e78b3794-af88-46ff-8203-afa3663905ed",
            "name": "hammad Arbisoft",
            "slug": "default",
            "active": true,
            "site": {
                "domain": "example.com",
                "name": "example.com"
            },
            "enable_data_sharing_consent": true,
            "enforce_data_sharing_consent": "at_enrollment",
            "branding_configuration": null,
            "enterprise_customer_entitlements": [],
            "identity_provider": null,
            "enable_audit_enrollment": false,
            "replace_sensitive_sso_username": false,
            "enable_portal_code_management_screen": false,
            "sync_learner_profile_data": false,
            "enable_audit_data_reporting": true
        },
        {
            "uuid": "8ffb97ac-f242-411a-99b8-ded9a3514b43",
            "name": "Hammad ENT",
            "slug": "hammad-ent",
            "active": true,
            "site": {
                "domain": "example.com",
                "name": "example.com"
            },
            "enable_data_sharing_consent": false,
            "enforce_data_sharing_consent": "at_enrollment",
            "branding_configuration": null,
            "enterprise_customer_entitlements": [],
            "identity_provider": null,
            "enable_audit_enrollment": false,
            "replace_sensitive_sso_username": false,
            "enable_portal_code_management_screen": false,
            "sync_learner_profile_data": false,
            "enable_audit_data_reporting": false
        },
        {
            "uuid": "3ef2bf97-f52e-4e38-a940-7a8bd9c12417",
            "name": "hammad White ENT",
            "slug": "white-ent",
            "active": true,
            "site": {
                "domain": "cmeonline.local:18000",
                "name": "cmeonline.local:18000"
            },
            "enable_data_sharing_consent": false,
            "enforce_data_sharing_consent": "at_enrollment",
            "branding_configuration": null,
            "enterprise_customer_entitlements": [],
            "identity_provider": null,
            "enable_audit_enrollment": false,
            "replace_sensitive_sso_username": false,
            "enable_portal_code_management_screen": false,
            "sync_learner_profile_data": false,
            "enable_audit_data_reporting": false
        }
    ],
    "next": null,
    "start": 0,
    "previous": null
}
```
Our new `basic_list`endpoint is returning a response like this
```
[
    {
        "id": "e78b3794-af88-46ff-8203-afa3663905ed",
        "name": "hammad Arbisoft"
    },
    {
        "id": "8ffb97ac-f242-411a-99b8-ded9a3514b43",
        "name": "Hammad ENT"
    },
    {
        "id": "3ef2bf97-f52e-4e38-a940-7a8bd9c12417",
        "name": "hammad White ENT"
    }
]
```